lib/test.rb

```diff
@@ -1124,7 +994,6 @@ def run
       ensure
         cleanup_after
       end
-      check_results
```

Comment:
This line needs to stay. `run` should return a boolean that indicates whether there were no failures.